quarkus/admin/src/test/java/org/apache/polaris/admintool/BootstrapCommandTest.java

```diff
@@ -132,6 +132,45 @@ public void testBootstrapFromInvalidFile(QuarkusMainLauncher launcher) {
         .contains("Bootstrap encountered errors during operation.");
   }
 
+  @Test
+  @Launch(
+      value = {"bootstrap", "-r", "realm1", "-c", "realm1,client1d,s3cr3t", "--print-credentials"})
+  public void testPrintCredentials(LaunchResult result) {
+    assertThat(result.getOutput()).contains("Bootstrap completed successfully.");
+    assertThat(result.getOutput()).contains("realm: realm1 root principal credentials: client1d:");
```

Comment:
Why not assert that `s3cr3t` is printed?

Reply:
It's actually _not_ printed -- maybe it should be. Because of PR#801 the secret supplied becomes the secondary secret while the primary secret is printed.

Reply:
I would rather we just solve #801 rather than add special logic around printing the secondary secret here

Reply:
TIL. I think this PR is good to merge, then and we'll follow-up on #801